### DISCUSSION TITLE
Concurrent not working

### DISCUSSION THREAD

**Discussion by IRediTOTO** [triggerdotdev/trigger.dev#1084] [Q&A]
I have concurrency is 2 like this 
![image](https://github.com/triggerdotdev/trigger.dev/assets/15695252/664e9170-5322-4090-a8ad-a746da93884c)
But it not working, I call job many time and they still working in same time 
![image](https://github.com/triggerdotdev/trigger.dev/assets/15695252/20944344-735e-4770-b988-fdc7110eb784)

Or am I misunderstanding about concurrent running?

**Comment by matt-aitken** [comment]
@IRediTOTO sorry for the slow reply, we're not good at checking the GitHub discussions at the moment. The best place for support is [our Discord](https://trigger.dev/discord). Are you still experiencing this issue? That's definitely a bug in v2.
